xdsl/dialects/riscv.py

```diff
@@ -3673,6 +3673,15 @@ def _print_immediate_value(printer: Printer, immediate: AnyIntegerAttr | LabelAt
             printer.print_string_literal(immediate.data)
 
 
+class BitwiseHasCanonicalizationPatternsTrait(HasCanonicalisationPatternsTrait):
+    @classmethod
+    def get_canonicalization_patterns(cls) -> tuple[RewritePattern, ...]:
+        from xdsl.transforms.canonicalization_patterns.riscv import (
+            BitwiseAndByZero
+        )
+
+        return (BitwiseAndByZero(),)
```

Comment:
Nice, can you add this to the `AndOp` operation?
Otherwise, this won't be used by the canonicalizer at all.